uint16 address
uint16[] registers
uint8 slave
---
